cardano-constitution/cardano-constitution.cabal

```diff
@@ -74,13 +74,13 @@ library
 
   build-depends:
     , aeson
-    , base               >=4.9   && <5
+    , base               >=4.9 && <5
     , containers
     , filepath
-    , plutus-core        ^>=1.30
-    , plutus-ledger-api  ^>=1.30
-    , plutus-tx          ^>=1.30
-    , plutus-tx-plugin   ^>=1.30
+    , plutus-core
+    , plutus-ledger-api
+    , plutus-tx
+    , plutus-tx-plugin
     , regex-tdfa
     , safe
     , template-haskell
```

Comment:
Can u put back the version numbering?

Reply:
I guess the prepare-release script does pick and update those, right?

Reply:
No, I believe that buy not putting an explicit version boundary we're saying: the current version from this cabal project.

Reply:
CHaP might need the explicit bounds, restoring them.